datumaro/util/scope.py

```diff
@@ -88,6 +88,15 @@ def add(self, cm: ContextManager[T]) -> T:
 
         return self._stack.enter_context(cm)
 
+    def add_many(self, *cms: ContextManager[T]) -> Tuple[T, ...]:
```

Comment:
It's worth noting that in every non-test call of this, the argument is an unpacked sequence. So maybe it would be better to change the argument to accept a sequence to start with?

Reply:
Currently it is only used in a single context, so I can't say whether one variant is better than another. Maybe, we should just put `*` into `add()` to make it a "drop-in" replacement for `with`.

Reply:
They still have literally the same text.

Reply:
Removed this funcitonality